src/logging-utils.ts

```diff
@@ -1,5 +1,5 @@
 import { createProjectLogger, createModuleLogger } from '@metamask/utils';
 
-export const projectLogger = createProjectLogger('eth-block-tracker');
+export const projectLogger = createProjectLogger('@metamask/eth-block-tracker');
```

Comment:
We shouldn't need this one here; the `createProjectLogger` function adds a prefix, so you can show log messages for all MetaMask libraries by setting `DEBUG="metamask:*"`, or in this case you could show log messages for this library by setting `DEBUG="metamask:eth-block-tracker:*"`. So I feel like `@metamask/` would be redundant.